src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -262,8 +276,12 @@ def minimize_crash(self,
                                         max_total_time)
 
     additional_args.extend([
-        constants.MINIMIZE_CRASH_ARGUMENT, max_total_time_argument,
-        constants.EXACT_ARTIFACT_PATH_FLAG + output_path, testcase_path
+        constants.MINIMIZE_CRASH_ARGUMENT,
+        max_total_time_argument,
+        constants.EXACT_ARTIFACT_PATH_FLAG + output_path,
+        constants.ARTIFACT_PREFIX_FLAG +
+        self._normalize_artifact_prefix(artifact_prefix),
```

Comment:
What if artifact_prefix is None, it will cause exception i think seeing the first line in _normalize_artifact_prefix (checks endswith).